src/actions/ui.js

```diff
@@ -171,14 +171,14 @@ export function setProjectDirectoryRoot(newRoot: string) {
   return ({ dispatch, getState }: ThunkArgs) => {
     const curRoot = getProjectDirectoryRoot(getState());
     if (newRoot && curRoot) {
-      const temp1 = newRoot.replace(/\/+/g, "/").split("/");
-      const temp2 = curRoot
+      const tmpNew = newRoot.replace(/\/+/g, "/").split("/");
```

Comment:
nit: I was thinking more like newRootArr, or something. I generally try to avoid variable names like tmp. There's always better names which better communicate intent.

Reply:
but since it was unfortunately already called temp1, I guess I'm fine with it.

Reply:
I can quickly make the tweaks.